Examples/Swift/CustomViewController.swift

```diff
@@ -96,22 +96,21 @@ class CustomViewController: UIViewController, MGLMapViewDelegate, AVSpeechSynthe
     func updateRouteProgress(routeProgress: RouteProgress) {
         guard let step = routeProgress.currentLegProgress.upComingStep else { return }
 
-        if let direction = step.maneuverDirection {
-            switch direction {
-            case .slightRight:
-                self.arrowView.text = "↗️"
-            case .sharpRight, .right:
-                self.arrowView.text = "➡️"
-            case .slightLeft:
-                self.arrowView.text = "↖️"
-            case .sharpLeft, .left:
-                self.arrowView.text = "⬅️"
-            case .uTurn:
-                self.arrowView.text = "⤵️"
-            default:
-                self.arrowView.text = "⬆️"
-            }
+        switch step.maneuverDirection {
+        case .slightRight:
+            self.arrowView.text = "↗️"
+        case .sharpRight, .right:
+            self.arrowView.text = "➡️"
+        case .slightLeft:
+            self.arrowView.text = "↖️"
+        case .sharpLeft, .left:
+            self.arrowView.text = "⬅️"
+        case .uTurn:
+            self.arrowView.text = "⤵️"
+        default:
```

Comment:
There's a subtle edge-case behavioral change here -- if `step.maneuverDirection = nil`, the arrow text becomes "⬆️". 

That may be desirable, I just thought i should point this out.

Reply:
Yes, I think that’s fine. Our other option is ⏹. 😉